app.js

```diff
@@ -1,8 +1,18 @@
 const express = require('express');
 const app = express();
-const { getApi } = require('./controllers/api.controllers')
+const { getApi } = require('./controllers/api.controllers');
+const { getApiTopics } = require('./controllers/topics.controllers');
+
+app.use(express.json());
```

Comment:
You aren't posting or patching right now so this isn't needed.

Reply:
erased.